ansible/roles/ghostunnel/templates/etc/systemd/system/ghostunnel.service.j2

```diff
@@ -0,0 +1,25 @@
+{# Copyright (C) 2021 Pedro Luis Lopez <pedroluis.lopezsanchez@gmail.com>
+ # Copyright (C) 2021 DebOps <https://debops.org/>
+ # SPDX-License-Identifier: GPL-3.0-only
+ #}
+# {{ ansible_managed }}
+
+# ghostunnel-{{ item.name }}.service unit file
+# Copyright (C) 2021 Pedro Luis Lopez <pedroluis.lopezsanchez@gmail.com>
+# Copyright (C) DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Remove the duplicated copyright and license information here.

Reply:
Done!